src/core/core.controller.js

```diff
@@ -76,26 +76,39 @@ module.exports = function(Chart) {
 		},
 
 		resize: function resize(silent) {
```

Comment:
A side note: should we remove all function names when it's a member ```resize: function()```, it would save a few bits.